### ISSUE TITLE
set pause to layer

### ISSUE THREAD

**Issue by rbjoensen** [mainsail-crew/MainsailOS#238]
### What happened

Klipper - Mainsailos

After installation, it has a defined macro "set pause to layer" but it does nothing. Some ideas?
The idea is that it should run a macro with FILAMENT_SWAP so that I don't have to think about it in the slicer.

### What did you expect to happen

Marco to run at layer

### How to reproduce

Try

**Comment by zellneralex** [comment]
Yes read the slicer section https://docs.mainsail.xyz/overview/slicer you need to prepare your gcode
